ext/reflection/php_reflection.stub.php

```diff
@@ -569,6 +569,10 @@ public function hasHook(PropertyHookType $type): bool {}
     public function getHook(PropertyHookType $type): ?ReflectionMethod {}
 
     public function isFinal(): bool {}
+
+    public function isReadable(?string $scope = 'static'): bool {}
+
+    public function isWritable(object $object, ?string $scope = 'static'): bool {}
```

Comment:
I think that object should be a nullable optional second parameter, because a property might be static. Also non-readonly properties can be checked without an object.

Reply:
Good point.